src/ucs/config/global_opts.c

```diff
@@ -23,6 +23,8 @@ ucs_global_opts_t ucs_global_opts = {
     .log_component         = {UCS_LOG_LEVEL_WARN, "UCX"},
     .log_print_enable      = 0,
     .log_file              = "",
+    .log_file_max_size     = SIZE_MAX,
+    .log_file_backup_count = 0,
```

Comment:
since log rotate is well known concept https://linux.die.net/man/8/logrotate
maybe name it 
UCX_LOG_FILE_SIZE=inf , log_file_size
UCX_LOG_FILE_ROTATE=0,  log_file_rotate